.github/workflows/release.yaml

```diff
@@ -24,6 +24,49 @@ jobs:
               }
             })
 
+  build-prqlc:
+    runs-on: ${{ matrix.os }}
+    env:
+      ARTIFACT_BASE_NAME:
+        prqlc-v${{ github.ref_type == 'tag' && github.ref_name || 0 }}-${{
+        matrix.target }}
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+          - os: ubuntu-latest
+            target: aarch64-unknown-linux-musl
+          - os: macos-latest
+            target: x86_64-apple-darwin
+          - os: macos-latest
+            target: aarch64-apple-darwin
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
+    steps:
+      - name: 📂 Checkout code
+        uses: actions/checkout@v3
+      - uses: ./.github/actions/build-prqlc
+        with:
+          target: ${{ matrix.target }}
+      - name: Create artifact for Linux and macOS
+        if: runner.os != 'Windows'
+        run: |
+          echo "ARTIFACT_NAME=${{ env.ARTIFACT_BASE_NAME }}.tar.gz" >>"$GITHUB_ENV"
```

Comment:
Thanks!

Sorry, but I ran it on my fork and it is clear that this does not work. I will fix it based on your suggestion.

![image](https://github.com/PRQL/prql/assets/50911393/a9899b05-5182-44bd-bbd0-cffd781ced79)

The next version release should be 0.9.0 because of the breaking changes and I think we need to resolve #2581 before the release, I think.
(I will of course work on this PR so this can be merged before release)